src/corelibs/U2Designer/src/PropertyWidget.cpp

```diff
@@ -433,42 +432,19 @@ QVariant ComboBoxWithChecksWidget::value() {
 }
 
 void ComboBoxWithChecksWidget::setValue(const QVariant &value) {
-    disconnect(cm, SIGNAL(itemChanged(const QModelIndex &)), this, SLOT(sl_itemChanged(const QModelIndex &)));
-    QStringList curList = value.toString().split(",");
-    if (cm == NULL) {
-        cm = new QStandardItemModel(items.size(), 1, comboBox);
-    } else {
-        cm->clear();
-        delete cm;
-        cm = new QStandardItemModel(items.size(), 1, comboBox);
+    if (cm == nullptr) {
```

Comment:
This field is assigned in the constructor (initModelView method) and is never changed. Why do you want to check it for nullptr here?